tests/conformance/test_conformance.py

```diff
@@ -356,94 +549,95 @@ def _delete_retry_test(host, id):
-                    warnings.warn(
-                        "Error creating retry test for {}: {}".format(method_name, e),
-                        UserWarning,
-                        stacklevel=1,
-                    )
-                    continue
-
-                # Populate resources.
-                try:
-                    resources = _populate_resources(client, json_resources)
-                except Exception as e:
-                    warnings.warn(
-                        "Error populating resources for {}: {}".format(method_name, e),
-                        UserWarning,
-                        stacklevel=1,
-                    )
-                    continue
-
-                # Run retry tests on library methods.
-                try:
-                    _run_retry_test(
-                        host, id, function, precondition_provided, **resources
-                    )
-                except Exception as e:
-                    # Should we be catching specific exceptions
-                    print(e)
-                    success_results = False
-                else:
-                    success_results = True
-
-                # Assert expected success for each scenario.
-                assert expect_success == success_results
-
-                # Verify that all instructions were used up during the test
-                # (indicates that the client sent the correct requests).
-                try:
-                    status_response = _check_retry_test(host, id)
-                    assert status_response["completed"] is True
-                except Exception as e:
-                    warnings.warn(
-                        "Error checking retry test status for {}: {}".format(
-                            method_name, e
-                        ),
-                        UserWarning,
-                        stacklevel=1,
-                    )
-
-                # Clean up and close out test in emulator.
-                try:
-                    _delete_retry_test(host, id)
-                except Exception as e:
-                    warnings.warn(
-                        "Error deleting retry test for {}: {}".format(method_name, e),
-                        UserWarning,
-                        stacklevel=1,
-                    )
+    scenario = _CONFORMANCE_TESTS[scenario_id - 1]
+    expect_success = scenario["expectSuccess"]
+    precondition_provided = scenario["preconditionProvided"]
+    json_resources = method["resources"]
+    method_name = method["name"]
+    instructions = case["instructions"]
+
+    if method_name not in method_mapping:
+        pytest.skip("No tests for operation {}".format(method_name),)
+
+    for function in method_mapping[method_name]:
+        # Create the retry test in the emulator to handle instructions.
+        try:
+            r = _create_retry_test(host, method_name, instructions)
+            id = r["id"]
+        except Exception as e:
+            warnings.warn(
```

Comment:
Again, this and following should be hard failures (for the specific case) rather than warnings. We can discuss the path from here given that there may still be some compliance gaps.

Reply:
Done, thanks!